cnb_image.go

```diff
@@ -346,9 +333,6 @@ func (i *CNBImageCore) AddLayerWithDiffIDAndHistory(path, _ string, history v1.H
 }
 
 func (i *CNBImageCore) Rebase(baseTopLayerDiffID string, withNewBase Image) error {
-	if i.Kind() != withNewBase.Kind() {
```

Comment:
If my underlying image i is a remote image, does it mean I can rebase it with a local or layout image?